src/effects/VST/VSTEffect.cpp

```diff
@@ -2204,7 +2221,7 @@ void VSTEffect::callSetProgram(int index)
 
    callDispatcher(effSetProgram, 0, index, NULL, 0.0);
    for (const auto &slave : mSlaves)
-      slave->callSetProgram(index);
+      slave->callSetProgramB(index);
```

Comment:
Can the distinction between `callSetProgram` and `callSetProgramB` now be eliminated?  I think so, but it needs some longer explanation from me.

Reply:
The short answer is: not, when doing realtime processing, at least at this stage (I mean, step 4)

Why? I tried the following: 
1) place breakpoint in CallSetProgram
2) choose a factory preset: when we reach CallSetProgram, mSlaves is empty

So if we did only destructive processing, then we could go with callSetProgramB only (i.e. the version which ignores slaves)

but:
if I start realtime processing and then choose a factory preset, when I reach callSetProgram, mSlaves is not empty - it has one item and I guess it is the item doing the realtime processing.